pandas/core/frame.py

```diff
@@ -8971,9 +8971,8 @@ def mode(self, axis=0, numeric_only=False, dropna=True) -> DataFrame:
         ostrich       bird     2    NaN
 
         By default, missing values are not considered, and the mode of wings
-        are both 0 and 2. The second row of species and legs contains ``NaN``,
-        because they have only one mode, but the DataFrame has two rows.
-
+        are both 0 and 2. Because the resulting DataFrame has two rows,
+        the second row of ``species`` and ``legs`` contains ``NaN``.
         >>> df.mode()
```

Comment:
```suggestion
        are both 0 and 2. Because the resulting DataFrame has two rows,
        the second row of ``species`` and ``legs`` contains ``NaN``.

        >>> df.mode()
```